crates/bevy_render/src/renderer/mod.rs

```diff
@@ -97,13 +101,29 @@ pub struct RenderInstance(pub Instance);
 #[derive(Resource, Clone, Deref, DerefMut)]
 pub struct RenderAdapterInfo(pub AdapterInfo);
 
+/// The texture format used by the renderer.
+/// Initially it's the first texture format in `AvailableTextureFormats`
+#[derive(Resource, Clone, Deref, DerefMut)]
+pub struct RenderTextureFormat(pub wgpu::TextureFormat);
+
+/// Available texture formats on a given platform. Will be inserted
```

Comment:
This needs to link back and forth with the struct above. It's easy to figure out their relation in the source, but many users will be interacting with this through pure docs.

Reply:
Done.